### ISSUE TITLE
Improve help text for "link text" on download files

### ISSUE THREAD

**Issue by kevwalsh** [department-of-veterans-affairs/va.gov-cms#3443]
## Description

The "Link text" field label and help text should be improved based on suggestions from @jenstrickland, and our UX writer. 

![Create_Benefits_Detail_Page___VA_gov_CMS](https://user-images.githubusercontent.com/643678/98629689-6e678580-22e7-11eb-85aa-4353391e3951.jpg)

## Acceptance Criteria
- [x] Go to /node/add/page and add a "LINK TO A FILE OR VIDEO" content block. 
- [x] Help text is improved, based on AC agreed to with @jenstrickland

**Comment by jenstrickland** [comment]
It would be helpful if the help text clarified that it is unnecessary to start the link text with the word "Download" or to indicate the file type. For example, if the text entered is "VA Form 10-10CG" and the file uploaded is a PDF, the system will display the icon (equivalent to Download) "VA Form 10-10CG (PDF)". If the file uploaded is a Welcome  Kit and it is a PPT, the system will display the icon (equivalent to Download) "Welcome Kit (PPT)".

**Comment by rachel-kauff** [comment]
@oksana-c I've mocked up my recommendations for the help text here, along with recommendations for corresponding changes to the paragraphs browser.

**Comment by rachel-kauff** [comment]
Screen Shot 2020-12-31 at 1.49.04 PM
Screen Shot 2020-12-31 at 1.49.14 PM

**Comment by rachel-kauff** [comment]
(purple indicates where I've made changes). I'll also attach a file for the new image for the paragraphs browser shortly.

**Comment by rachel-kauff** [comment]
Sized file for new paragraphs browser thumbnail: 
![Artboard – 1](https://user-images.githubusercontent.com/41017988/103431195-4676fe80-4b81-11eb-88de-cabe26535a6d.png)

**Comment by rachel-kauff** [comment]
Update update: I *think* I managed to successfully make the changes in config.

**Comment by oksana-c** [comment]
good morning @rachel-kauff . The “Download link” paragraph is also used for linking to Videos on several node published on production, so the help text will need to account for Video option.

**Comment by oksana-c** [comment]
@rachel-kauff , I just found another related issue that may affect your final recommendation for this paragraph -  https://github.com/department-of-veterans-affairs/va.gov-cms/issues/3231

**Comment by rachel-kauff** [comment]
Update: I looked in to the issues Kev raised in #3231 and made some comments/ recommendations there. tldr: Addressing this well will require a little more than just some help text.

**Comment by rachel-kauff** [comment]
Update: 
*Deliverables*
The deliverables for the interim solution (uses the same paragraph for both video links and file downloads). 
-  Microcopy for linked text field label and help text
   - Mockup: ![Screen Shot 2021-01-28 at 4.02.04 PM.png](https://images.zenhubusercontent.com/5e399cf5446c5b4ca0e31e42/49ec3d07-95f3-49e4-8d43-0e1addc48e14)
- Paragraphs browser thumbnail image file
   - SVG: ![Paragraphs-browser-thumbnail.svg](https://images.zenhubusercontent.com/5e399cf5446c5b4ca0e31e42/18e3d99e-48a9-46ab-85a9-74c86d3cc136)
   - PNG: ![Paragraphs-browser-thumbnail@2x.png](https://images.zenhubusercontent.com/5e399cf5446c5b4ca0e31e42/97d43066-13f0-47fd-bf8e-74ec29e5f57c)
- Microcopy for paragraphs browser - label and description
    - Mockup: ![Screen Shot 2021-01-28 at 4.03.57 PM.png](https://images.zenhubusercontent.com/5e399cf5446c5b4ca0e31e42/8521cb99-3afa-44d1-9728-d06b820392ae)

*Artifacts*
- [Prototype file for MVP deliverables](https://www.figma.com/file/6AWSPB0ijeoHQFToF2qsRo/Rachel-Sprint-24?node-id=42%3A2) (Figma)
- [Prototype file with post-MVP proposals](https://balsamiq.cloud/s21kwcm/py46j7t/r9034) (Balsamiq)

**Comment by oksana-c** [comment]
thank you @rachel-kauff ! this is a perfect short term improvement. I opened a follow up to plug this into config here https://github.com/department-of-veterans-affairs/va.gov-cms/issues/4133
